src/NUnitFramework/tests/Attributes/TestOrderAttributeTests.cs

```diff
@@ -27,18 +26,15 @@ public void CheckOrderIsCorrect()
 
         [Test]
         [TestCaseSource(nameof(Cases))]
-        public void CheckClassOrderIsCorrect(List<Type> candidateTypes)
+        public void CheckClassOrderIsCorrect(Type[] candidateTypes)
         {
-            var testSuite = TestBuilder.MakeFixture(candidateTypes);
+            var testSuite = new TestSuite("dummy").Containing(candidateTypes);
 
-            var work = TestBuilder.PrepareWorkItem(testSuite, null) as CompositeWorkItem;
+            var work = TestBuilder.CreateWorkItem(testSuite) as CompositeWorkItem;
```

Comment:
Correct me if I'm wrong but my memory is that the return type of `TestBuilder.CreateWorkItem` is already `CompositeWorkItem` s othe `as` is redundant?

Reply:
It returns `WorkItem` in this overload. But I'll look at the overloads again.

Reply:
Taking another look, this does make sense. Though again I think the safe cast clouds intent, whichever way you look at it. But nbd.